js/script.js

```diff
@@ -0,0 +1,61 @@
+window.addEventListener('DOMContentLoaded', function() {
+
+//Navbar
+const barsButton = document.querySelector('.header__barButtonIcon');
+const headerNav = document.querySelector('.header__nav');
+
+
+barsButton.addEventListener('click', () => {
+    headerNav.classList.toggle('nav--active')
+})
+
+//Scroll
+function toScroll(PosTop, navH) {
```

Comment:
`navH` mało mówi :) nie bójmy się pisać pełnymi słowa. Skróty są fajne jeśli sami piszemy kod i mamy dobrą pamięć. Za 3 msc nikt może nie wiedzieć czym jest `navH` i będziemy musieli sprawdzać. Może np: `navHeight` to samo tyczy się `posTop` -> `positionTop`